src/ontology/external/mondo-omim-genes.robot.tsv

```diff
@@ -2353,7 +2353,6 @@ http://purl.obolibrary.org/obo/MONDO_0013259	http://identifiers.org/hgnc/10013	O
 http://purl.obolibrary.org/obo/MONDO_0013263	http://identifiers.org/hgnc/34383	OMIM:613428	https://omim.org/entry/613425
 http://purl.obolibrary.org/obo/MONDO_0013264	http://identifiers.org/hgnc/17142	OMIM:613435	https://omim.org/entry/602432
 http://purl.obolibrary.org/obo/MONDO_0013268	http://identifiers.org/hgnc/450	OMIM:613451	https://omim.org/entry/605420
-http://purl.obolibrary.org/obo/MONDO_0013270	http://identifiers.org/hgnc/3811	OMIM:613454	https://omim.org/entry/164874
```

Comment:
MONDO_0013270 is obsolete in Mondo

Reply:
Hmm. The OMIM pipeline doesn't have access to see which Mondo terms are deprecated. It will look for anything inside of these files `mondo_exactmatch_omim.sssom.owl` and `mondo_exactmatch_omimps.sssom.owl`, which it downloads from the Mondo master branch.

I don't think it would be easy to filter deprecated in the OMIM repo, but `mondo-ingeset` is set up much more easily to do that.

Can you make a ticket for this?

Reply:
@joeflack4 why is a ticket needed? Seeing this line removed is correct.

Reply:
Oh sorry it is red not green. I was focusing more on your words -_- / I figured that the comment was an indication of an issue, rather than showing things were working as expected